pkg/metrics/constants.go

```diff
@@ -66,3 +66,9 @@ func Measure(observer prometheus.Observer) func() {
 	start := time.Now()
 	return func() { observer.Observe(time.Since(start).Seconds()) }
 }
+
+// NodeClaimTerminationDurationBuckets returns a []float64 of default threshold values for nodeClaim termination duration histograms.
+func NodeClaimTerminationDurationBuckets() []float64 {
```

Comment:
More for my understanding, how did you determine these duration buckets?

Reply:
Tested this on my local cluster